src/auto/accel_label.rs

```diff
@@ -53,9 +53,9 @@ pub trait AccelLabelExt: 'static {
 
     fn set_accel(&self, accelerator_key: u32, accelerator_mods: gdk::ModifierType);
 
-    fn set_accel_closure<'a, P: Into<Option<&'a glib::Closure>>>(&self, accel_closure: P);
+    fn set_accel_closure(&self, accel_closure: Option<&glib::Closure>);
 
-    fn set_accel_widget<'a, P: IsA<Widget> + 'a, Q: Into<Option<&'a P>>>(&self, accel_widget: Q);
+    fn set_accel_widget<P: IsA<Widget>>(&self, accel_widget: &P);
```

Comment:
That should be `Option<&P>`, not `&P`. The PR is invalid. :)

Reply:
Hm, this still not fixed, seems update go wrong as it have right in my regen.

Reply:
I think my rebase went crazy. I'll restart the whole process.